src/bsoncxx/CMakeLists.txt

```diff
@@ -160,8 +160,8 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
     FILES_MATCHING
       PATTERN "*.h"
       PATTERN "*.hpp"
-      REGEX ".+/private" EXCLUDE
-      REGEX ".+/test" EXCLUDE
+      REGEX "${CMAKE_CURRENT_SOURCE_DIR}/.*private" EXCLUDE
```

Comment:
I'll try `\Q` and see what happens.

I remove the slash before private because we want to catch both top-level and sublevel private.  Possibly we need two regexes for those cases:

```
REGEX "${CMAKE_CURRENT_SOURCE_DIR}/private" EXCLUDE
REGEX "${CMAKE_CURRENT_SOURCE_DIR}/.*/private" EXCLUDE
```